src/main/java/org/usfirst/frc4904/robot/subsystems/Elevator.java

```diff
@@ -0,0 +1,60 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import org.usfirst.frc4904.standard.commands.motor.MotorIdle;
+import org.usfirst.frc4904.standard.custom.Overridable;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.MotionController;
+import org.usfirst.frc4904.standard.custom.sensors.CANEncoder;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.standard.subsystems.motor.PositionSensorMotor;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Elevator extends PositionSensorMotor implements Overridable {
+	public static final double UP_SPEED = 0.5;
+	public static final double DOWN_SPEED = -0.5;
+	public static final double ENCODER_TICKS = 1024;
+	public static final double DISTANCE_PER_ROTATION = 0; //SET LATER
+	public static final double TICK_MULTIPLIER = DISTANCE_PER_ROTATION;
+	public static final double MIN_POSITION = 0;
+	public static final double MAX_POSITION = 500; // ARBITRATY NUMBER, FIND OUT LATER
+	public final CANEncoder elevEncoder;
+	public ElevState state = ElevState.UP;
```

Comment:
Why does the elevator's state start up and not down?

Reply:
check with design, there might be a reason